xbmc/ServiceManager.cpp

```diff
@@ -138,38 +141,57 @@ bool CServiceManager::StartAudioEngine()
   return m_ActiveAE->Initialize();
 }
 
-bool CServiceManager::Init3()
+bool CServiceManager::InitStageThree()
 {
+  m_peripherals.reset(new PERIPHERALS::CPeripherals(*m_announcementManager));
   m_peripherals->Initialise();
-  m_PVRManager->Init();
```

Comment:
peripherals must not be in stage 2, because our test suite inits stage 2 and doesn't provide enough for the peripherals subsystem